src/components/select/filterable-select/filterable-select.mdx

```diff
@@ -165,6 +165,10 @@ This allows use-cases like server-side filtering of options, or rich formatting
 
 <Canvas of={FilterableSelectStories.CustomFilterAndOptionStyle} />
 
+## Testing with RTL
```

Comment:
**suggestion(non-blocking)**: what do you think about including the full library name, since this is the first instance its used on the page? 

This might improve clarity for developers who aren't aware of the library's acronym, and avoid confusion it with [Right to Left script](https://developer.mozilla.org/en-US/docs/Glossary/RTL).

```suggestion
## Testing with React Testing Library (RTL)
```